conf/distro/rpb-qt5.conf

```diff
@@ -0,0 +1,8 @@
+require conf/distro/include/rpb.inc
+
+DISTRO_NAME = "Reference-Platform-Build-QT5"
+
+PACKAGECONFIG_append_pn-qtbase = " gles2 eglfs fontconfig kms"
```

Comment:
yes, i confirm that we need to set gles2 and fontconfig in rpb.inc and only set eglfs and kms in rpb-qt5.conf. with that change we can build the Qt5 image for X11 and for wayland too.